cmd/opts/opts.go

```diff
@@ -949,6 +952,20 @@ type RecreateOpts struct {
 	cmd
 }
 
+type PcapOpts struct {
+	Args AllOrInstanceGroupOrInstanceSlugArgs `positional-args:"true"`
+
+	Interface   string        `long:"interface" short:"i" description:"Specifies the network interface to listen on." default:"eth0" required:"false"`
+	Filter      string        `long:"filter" short:"f" description:"Filter to apply when running tcpdump."`
+	SnapLength  uint32        `long:"snaplen" short:"s" description:"Snarf snaplen bytes of data from each packet rather than the default of 65535 bytes." default:"65535"`
+	Output      string        `long:"output" short:"o" description:"File to write pcap to."`
```

Comment:
Currently, if `--output` is not provided to the command, it will connect to VMs and attempt to do the capture, only to fail with `write to output file failed: open : no such file or directory`. It may be better to do some upfront validation that it has been provided and error earlier, similar to what is being done with interface and snap length. Alternatively, you could provide a default output file name if there is one that would make sense.

Reply:
I marked `output` as required argument in the command. You are right, it does not make sense to write pcap without output file.